python/ray/util/client_connect.py

```diff
@@ -15,7 +15,7 @@
 def connect(
     conn_str: str,
     secure: bool = False,
-    metadata: List[Tuple[str, str]] = None,
+    _metadata: List[Tuple[str, str]] = None,
```

Comment:
Final nit, can we leave this signature the same?

Reply:
@ijrsvt 
Of course! I changed `connect` to take metadata.

Just to make sure this looks okay:

clinet_builder.py
```python
        client_info_dict = ray.util.client_connect.connect(
            self.address,
            job_config=self._job_config,
            _credentials=self._credentials,
            ray_init_kwargs=self._remote_init_kwargs,
            metadata=self._metadata,
        )
```

Do we think this looks good to go?